packages/simcore-sdk/tests/unit/test_node_ports_v2_port.py

```diff
@@ -319,7 +318,7 @@ class PortParams(NamedTuple):
                 exp_new_value=FileLink(
                     store=simcore_store_id(),
                     path=f"{project_id()}/{node_uuid()}/no_file/{this_node_file_name().name}",
-                    e_tag=e_tag(),
+                    eTag=e_tag(),
```

Comment:
Ohh boy this one looks dangerous to me. Since this model is used in other services. Might break compatibility with old code. Can you please make sure the field is called `e_tag` and not `eTag`.

Reply:
What I mean, is it has to work with the previous testing code.

Reply:
Yup, you're right